tasks/docs/build.js

```diff
@@ -35,7 +36,8 @@ var
 
 module.exports = function (callback) {
     // use a different config
-    config = configSetup.addDerivedValues(config);
+    let config = extend(true, {}, configDocs);
```

Comment:
Why so complicated rewrite? If config cannot be a const, please make it a let instead.